R/shiny.R

```diff
@@ -134,16 +134,21 @@ run <- function(file = "index.Rmd", dir = dirname(file), default_file = NULL,
   yaml_front <- if (length(target_file)) yaml_front_matter(target_file)
   runtime <- yaml_front$runtime
   theme <- render_args$output_options$theme
+  # Let shiny::getCurrentTheme() know about the yaml's theme, so
+  # things like `bslib::bs_themer()` can work with prerendered documents.
+  # Also note that we add the actual shiny::bootstrapLib() dependency
+  # inside the document's pre-processing hook so the 'last' version of
+  # the theme wins out
   if (length(target_file)) {
     format <- output_format_from_yaml_front_matter(read_utf8(target_file))
```

Comment:
@cpsievert Regarding my question in https://github.com/rstudio/rmarkdown/pull/2049/files#r586418699, I wonder if `render_args$output_options` should not be passed to this function

I believe this should be 
```suggestion
format <- output_format_from_yaml_front_matter(read_utf8(target_file), render_args$output_options)
```

This way if theme provided in `render_args$output_options` in `rmarkdown::run` is will correctly be merged with any other option set in YAML header.

I missed that at the first review but it is my understanding now. 

I want to confirm the intention in https://github.com/rstudio/rmarkdown/pull/2049/files#r586418699 before doing any modification. Thanks.

Reply:
Ahh, yes, thanks for catching that! If you happen to include a fix in another PR, feel free to tag me on it

Reply:
Ok. So I'll push a fix for that then. I believe we can remove the line above also.

Reply:
See #2065